controller/node_controller.go

```diff
@@ -355,13 +356,22 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 			diskStatus.StorageAvailable = diskInfo.StorageAvailable
 		}
 
+		condition := types.GetNodeCondition(diskConditions, types.NodeConditionTypeDiskSchedulable)
+		condition.LastTransitionTime = util.Now()
+		condition.LastProbeTime = util.Now()
 		// check disk pressure
 		if diskInfo.StorageAvailable <= disk.StorageMaximum*minimalAvailablePercentage/100 {
-			diskStatus.State = types.DiskStateUnschedulable
+			condition.Status = types.ConditionStatusFalse
+			condition.Reason = string(types.NodeConditionReasonDiskUnschedulable)
```

Comment:
The reason is not `Unschedulable`. It's `DiskPressure`. `Unschedulable` is the result of `DiskPressure`.